coordinator/src/notifications.rs

```diff
@@ -115,20 +115,22 @@ fn build_notification(kind: &NotificationKind) -> fcm::Notification<'_> {
     let mut notification_builder = fcm::NotificationBuilder::new();
     match kind {
         NotificationKind::PositionSoonToExpire => {
-            notification_builder.title("Your position is about to expire");
-            notification_builder.body("Rollover your position for the next cycle.");
+            notification_builder.title("Your position is about to expire ⏳");
+            notification_builder
+                .body("Open your app to rollover your position for the next cycle.");
         }
         NotificationKind::PositionExpired => {
-            notification_builder.title("Your position has expired");
-            notification_builder.body("Close your position.");
+            notification_builder.title("Your position has expired 🥴");
+            notification_builder.body("Open your app to execute the expiration.");
         }
         NotificationKind::PositionLiquidated => {
-            notification_builder.title("Your position has been liquidated");
-            notification_builder.body("Close your position.");
+            notification_builder.title("Your position has been liquidated 📉📈");
```

Comment:
The title doesn't seem to fit in the screenshot you took.

Reply:
yeah, should we remove the emojis?

Reply:
What about: "Woops, someone got liquidated 💸"

Reply:
We could probably make a specific notification depending on wether the user or the coordinator got liquidated.

Reply:
Thanks for the feedback, I will work on this tomorrow.